openslides/core/static/js/core/start.js

```diff
@@ -4,31 +4,55 @@
 
 angular.module('OpenSlidesApp.core.start', [])
 
-.run([
+.factory('OS', [
```

Comment:
Nice. But I would prefer to use something more explicit like `OpenSlidesStart` or thelike.